fe/fe-core/src/main/java/com/starrocks/sql/analyzer/ShowStmtAnalyzer.java

```diff
@@ -70,6 +71,15 @@ public Void visitShowMaterializedViewStmt(ShowMaterializedViewStmt node, Connect
             return null;
         }
 
+        @Override
+        public Void visitShowCreateTableStmt(ShowCreateTableStmt node, ConnectContext context) {
+            if (node.getTbl() == null) {
+                ErrorReport.reportSemanticException(ErrorCode.ERR_NO_TABLES_USED);
+            }
+            node.getTbl().normalization(context);
```

Comment:
is it neccessary to check whether db and table exist here?

Reply:
handleShowCreateTable has checked whether db and table exist